src/modules/task.js

```diff
@@ -1,62 +1,75 @@
 class TodoList {
   constructor() {
-    this.taskLiskArr = JSON.parse(localStorage.getItem('taskstore')) || [];
+    this.taskArr = JSON.parse(localStorage.getItem('taskstore')) || [];
   }
 
-  addTask = (item) => {
-    this.taskLiskArr.push({
-      description: item, completed: false, index: this.taskLiskArr.length + 1,
-    });
-    this.populateLocalStorage();
-    this.displayTask();
-  }
+    addTask = (item) => {
+      this.taskArr.push({ description: item, completed: false, index: this.taskArr.length + 1 });
+      this.populateLocalStorage();
+      this.displayTask();
+    }
 
-  removeTask = (index) => {
-    this.taskLiskArr = this.taskLiskArr.filter((task) => task.index !== index);
-    this.updateIndex();
-    this.populateLocalStorage();
-    this.displayTask();
-  }
+    removeTask = (index) => {
+      this.taskArr = this.taskArr.filter((task) => task.index !== index);
+      this.updateIndex();
+      this.populateLocalStorage();
+      this.displayTask();
+    }
 
-  updateIndex = () => {
-    this.taskLiskArr = this.taskLiskArr.map((task, index) => {
-      task.index = index + 1;
-      return task;
-    });
-  }
+    updateIndex = () => {
+      this.taskArr = this.taskArr.map((task, index) => {
+        task.index = index + 1;
+        return task;
+      });
+    }
 
- displayTask = () => {
-   const todo = document.querySelector('#toDoInputList');
-   todo.innerHTML = '';
-   for (let i = 0; i < this.taskLiskArr.length; i += 1) {
-     todo.innerHTML += `<div class='todo-div'>
-        <input type='checkbox' class='check' id="${i}">
-        <li class='todoListItem' contenteditable="true"
-        >${this.taskLiskArr[i].description}</li>
-        <div class="icons"
-        <i class="fa-solid fa-ellipsis-vertical"></i>
-        <i class="fa-solid fa-trash-can" data-index=${i + 1}></i>
-        </div>
-        </div>
-        `;
-   }
-   const deleteBtn = document.querySelectorAll('.fa-trash-can');
-   deleteBtn.forEach((button) => {
-     button.addEventListener('click', (event) => {
-       const id = parseInt(event.target.getAttribute('data-index'), 10);
-       this.removeTask(id);
+   displayTask = () => {
+     const todo = document.querySelector('.todo-trash');
+     todo.innerHTML = '';
+     for (let i = 0; i < this.taskArr.length; i += 1) {
+       todo.innerHTML += `
+          <div class="todo-div">
+          <div class="todos">
+          <input type="checkbox" name="" id="${i}">
+          <p contenteditable=>${this.taskArr[i].description}</p>
```

Comment:
- [ ] [OPTIONAL] I tried to edit the task content and it was working, great job 👏 👏. However, when I refreshed the page the changes I made were not there. I think it is because they are not updated in the local storage. It would be good if you add functionality to update it in the local storage. 😃 😎.